nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/trash/test/TestTrashBackwardMechanism.java

```diff
@@ -1,5 +1,5 @@
 /*
- * (C) Copyright 2018 Nuxeo (http://nuxeo.com/) and others.
+ * (C) Copyright 2018-2019 Nuxeo (http://nuxeo.com/) and others.
```

Comment:
this modification is not related to the fix, is it right ? i see only the modification of head

Reply:
Yes it's a forgotten discard